extern/storage-sealing/types.go

```diff
@@ -88,7 +88,7 @@ type SectorInfo struct {
 	PreCommitInfo    *miner.SectorPreCommitInfo
 	PreCommitDeposit big.Int
 	PreCommitMessage *cid.Cid
-	PreCommitTipSet  TipSetToken
```

Comment:
SectorInfo is an on-disk struct, and we probably shouldn't break it if we can.

(It's PreCommitTipSet instead of just height because we used to use it in lookups in later states, but that wasn't reorg-proof enough. Now it's here mostly for debugging)